### PR TITLE
 introductions of centroid() of ellipse

### PR DESCRIPTION
**Pull request by rushyam**
to fix the consistency issue.
polygon have centroid method.

### CODE REVIEW

**Comment by rushyam** [comment]
@normalhuman what is your suggestion?

File: sympy/geometry/ellipse.py [hunk 1]
```diff
@@ -1267,6 +1268,26 @@ def vradius(self):
...
+        Examples
+        ========
+        >>> from sympy import Point, Ellipse
+        >>> p1 = Point(0, 0)
+        >>> e1 = Ellipse(p1, 3, 1)
+        >>> e1.centroid
+        Point2D(0, 0)
+
+        """
+        return self.center
```
Comment (moorepants) [inline sympy/geometry/ellipse.py]: I've commented on this before. I think we should have either centroid or center, not both. There is not reason to be redundant.

Comment (rushyam) [reply sympy/geometry/ellipse.py]: @moorepants  yes,
Centre is used in the class itself almost everywhere. It is not good to remove the centre.

Comment (moorepants) [reply sympy/geometry/ellipse.py]: Ok, so we should stick with center. There is no need to add a centroid property.

**Comment by moorepants** [comment]
Closing due to redundancy. Feel free to reopen if there is a valid reason to have this property.

**Pull request closed without merging**